port/common/omrsysinfo_helpers.h

```diff
@@ -62,4 +62,6 @@ extern uint32_t
 omrsysinfo_get_cpu_extended_model(uint32_t processorSignature);
 #endif /* defined(OMR_OS_WINDOWS) || defined(J9X86) || defined(J9HAMMER) */
 
+extern double
+omrsysinfo_calculate_cpu_load(J9SysinfoCPUTime *new, J9SysinfoCPUTime *old);
```

Comment:
`new` is a less than ideal name to use here; I expect compile failures in C++ contexts (same comment for the linuxs390 header file).